### ISSUE TITLE
Get-HPOVDriveEnclosureInventory with -Assigned or -Available parameter

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#464]
Using parameter -Available, the cmdlet Get-HPOVDriveEnclosureInventory should return list of drives available for assignment. Instead it returns all drives (even some of them are already assigned)

```PowerShell
$res                            = Get-HPOVSasLogicalInterconnect | where name -eq  '3Frames-LIG-SAS-1'
write-host '=================== Query  Available disk drives'
Get-HPOVDriveEnclosureInventory -Available -SasLogicalInterconnect $res | where Capacity -eq 1864 | select name,DriveEnclosureName, logical*
write-host '=================== Query  Assigned disk drives'
Get-HPOVDriveEnclosureInventory -Assigned  -SasLogicalInterconnect $res | where Capacity -eq 1864 | select name,DriveEnclosureName, logical*
```
[2003110951_HPOV_transcript.log](https://github.com/HewlettPackard/POSH-HPOneView/files/4319591/2003110951_HPOV_transcript.log)

**Comment by ChrisLynchHPE** [comment]
This is a defect with the validation logic.  It will be fixed in the next library update.

**Comment by ChrisLynchHPE** [comment]
This is addressed in [Release 5.00.2368.2592](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.00.2368.2592) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.500/5.00.2368.2592).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
